components/LeftImageTextLayout.jsx

```diff
@@ -1,17 +1,37 @@
 /*
```

Comment:
when I run localhost I'm getting this sort of error:
```
Warning: Failed prop type: Invalid prop `image` of type `object` supplied to `Title`, expected `string`.
    at Title (webpack-internal:///./components/Title.jsx:53:5)
    at About
    at exports.ThemeProvider (/Users/sharonhe/Documents/PuMP/pump-web-v2/node_modules/styled-components/dist/styled-components.cjs.js:1:24021)
    at MyApp (webpack-internal:///./pages/_app.jsx:50:3)
    at ae (/Users/sharonhe/Documents/PuMP/pump-web-v2/node_modules/styled-components/dist/styled-components.cjs.js:1:13278)
    at AppContainer (/Users/sharonhe/Documents/PuMP/pump-web-v2/node_modules/next/dist/next-server/server/render.js:27:952)
```
    
   Not sure if you can take a look and see why?

Reply:
Yep, wrong proptype for Image in Title component (string instead of object)

Reply:
Ah makes sense. Honestly no one uses proptype right now but I'm glad you're using it because it adds a benefit of TypeScript